portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,94 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/*
+ In order to access the secret page you have to solve a randomly generated equation.
+ Upon the input of a correct answer a button will appear, linking the user to the page.
+ */
+function secretPage(){
```

Comment:
nit: Google style would have the curly brace separated from the parameter list by a space:
```suggestion
function secretPage() {
```